cmd/cmdutil/helpers.go

```diff
@@ -83,6 +94,36 @@ func ValidateArgs(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func ValidateMaxConcurrency(value string) bool {
```

Comment:
I decided to leverage the AWS Docs provided regex for validating --max-concurrency